http-server/src/test/java/org/triplea/server/moderator/toolbox/moderators/ModeratorsControllerTest.java

```diff
@@ -52,7 +53,7 @@ void getModerators() {
 
   @Test
   void isSuperModPositiveCase() {
-    when(authenticatedUser.isAdmin()).thenReturn(true);
+    when(authenticatedUser.getUserRole()).thenReturn(UserRole.ADMIN);
```

Comment:
Are "admin" and "super mod" the same thing?

Reply:
Yes; alignment of terms would be nice, but it's not in scope of this refactor. With the DB now updated to have roles, we can use ADMIN as a term.  Before we had an 'isAdmin' column in DB that meant moderator. Now that column is 'role', so we can use 'admin' without confusing 'admin' with 'moderator'. 'SuperMod' is only in the toolbox, which is still beta and we have not completely baked in the term 'super mod' just yet.

Reply:
Ok. I'm fine leaving it out of scope for now. It would be good long term for this overall project to get to pretty concise and consistent terminology so its a bit easier to understand.

Reply:
Generally agree; I think part of the problem is we don't have a domain term yet for 'admin', the role simply did not really exist. We could call it 'council member', but that does not capture the role well.

There are some glaring naming problems that do not match the domain:
- a player is called a node instead of as a player
- a 'game' refers to an XML when that is more a map. We have a 'select' map option, not a 'select game' option. Which 'maps' do you like, refers to specific XML variants, not to a set of XMLs

With that said, w.r.t ADMIN, does that seem reasonable? I think 'moderator' and 'player' are pretty clear. Anonymous is roughly well understood as opposed to registered or a 'player'. 'Super-mod' is something I made up when doing toolbox 2.0, largely to not collide with the 'isAdmin' DB column.